### PR TITLE
fix: ci

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "build:types": "rimraf ./types/* && tsc --declaration --emitDeclarationOnly --outDir types && node ./scripts/extend-webpack-types.js && prettier \"types/**/*.ts\" --write && prettier \"types/**/*.ts\" --write",
     "build": "npm-run-all -p \"build:**\"",
     "test:only": "jest",
-    "test:coverage": "npm run test:only -- --coverage",
+    "test:coverage": "npm run test:only -- --coverage --maxWorkers=2",
```
Comment (snitin315) [inline package.json:30]:
```suggestion
    "test:coverage": "npm run test:only -- --coverage --detectOpenHandles",
```

File: package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "build:types": "rimraf ./types/* && tsc --declaration --emitDeclarationOnly --outDir types && node ./scripts/extend-webpack-types.js && prettier \"types/**/*.ts\" --write && prettier \"types/**/*.ts\" --write",
     "build": "npm-run-all -p \"build:**\"",
     "test:only": "jest",
-    "test:coverage": "npm run test:only -- --coverage",
+    "test:coverage": "npm run test:only -- --coverage --detectOpenHandles",
```
Comment (snitin315) [inline package.json:30]:
```suggestion
    "test:coverage": "npm run test:only -- --coverage --forceExit",
```

**Comment by snitin315** [comment]
@alexander-akait this now seems to fix CI for windows.

All 3 consecutive attempts were green for all `windows` jobs.

- [Attempt 1](https://github.com/webpack/webpack-dev-server/actions/runs/1914971336/attempts/1)
- [Attempt 2](https://github.com/webpack/webpack-dev-server/actions/runs/1914971336/attempts/2)
- [Attempt 3](https://github.com/webpack/webpack-dev-server/actions/runs/1914971336/attempts/3)

**Review by alexander-akait — APPROVED** [review]
Good work, thanks!

**Pull request merged**